tests/e2e/p/l1.go

```diff
@@ -267,6 +280,36 @@ var _ = e2e.DescribePChain("[L1]", func() {
 					},
 				})
 			})
+
+			tc.By("fetching the subnet conversion attestation", func() {
+				unsignedSubnetConversion := must[*warp.UnsignedMessage](tc)(warp.NewUnsignedMessage(
```

Comment:
1. Yeah - I think it makes sense to factor `must` out in a separate PR.
2. The main reason `must` was added was so that initializing more complex types would read more easily.
 
I waffled on the usage of `must` in the case that you described above. _but_ in the case where we do:
```golang
value1, err := make1(
    must[myType](make0(...)),
)
require.NoError(t, err)
```
vs
```golang
value0, err := make0(...)
require.NoError(t, err)

value1, err := make1(value0)
require.NoError(t, err)
```
I think this is nicer - as it reads like nested struct creation (to me).

Not a hill I'll die on though. If other people find it more difficult to read, we can remove it.